docs/docs/install/terminfo.md

```diff
@@ -14,5 +14,12 @@ infocmp rio
 If it is not present already, you can install it globally with the following command:
 
 ```bash
```

Comment:
Great idea and initiative, a suggestion for this change will be

```suggestion
When cloned locally, from the root of the repository run:

```bash
sudo tic -xe rio misc/rio.terminfo
```

If has not cloned Rio locally

```bash
curl -o rio.terminfo https://raw.githubusercontent.com/raphamorim/rio/main/misc/rio.terminfo
sudo tic -xe rio rio.terminfo
rm rio.terminfo
```
```